hedera-node/test-clients/build.gradle.kts

```diff
@@ -315,6 +315,8 @@ val cleanYahCli =
         delete(File(project.file("yahcli"), "yahcli.jar"))
     }
 
+dependencies { implementation(project(":swirlds-merkledb")) }
```

Comment:
Is this necessary? I would think adding it to the module-info file would be sufficient

Reply:
It seems it's not needed anymore, I will remove it.